backend/app/testbot/main.py

```
@@ -0,0 +1,47 @@
+
+def main():
+    # Compute the base directory (i.e. testbot directory)
+    base_dir = os.path.dirname(os.path.abspath(__file__))
+    data_dir = os.path.join(base_dir, "chatbot_data")
+    
+    # Build KB for Chatbot A
+    chatbot_A_id = "chatbot_A"
+    dir_chatbot_A = os.path.join(data_dir, "chatbot_A")
+    data_paths_A = [
+        os.path.join(dir_chatbot_A, "products.md"),
+        os.path.join(dir_chatbot_A, "products.csv")
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)